examples/bpi-monolith-test-suite/bpi/bpi.ts

```diff
@@ -0,0 +1,136 @@
+import { Agreement } from "./components/storage/agreement";
+import { BpiSubject } from "./components/identity/bpiSubject";
+import { Invitation } from "./components/workgroup/invitation";
+import { Workgroup } from "./components/workgroup/workgroup";
+import { Workstep } from "./components/workgroup/workstep";
+import { BpiMessage } from "./components/messaging/bpiMessage";
+import { TransactionPoolComponent } from "./components/transactions/transactionPool";
+import { IMessagingComponent } from "./components/messaging/messaging.interface";
+import { IWorkgroupComponent } from "./components/workgroup/workgroup.interface";
+import { IIdentityComponent } from "./components/identity/identity.interface";
+import { ITransactionPoolComponent } from "./components/transactions/transactionPool.interface";
+import { IVirtualStateMachineComponent } from "./components/vsm/vsm.interface";
+import { VirtualStateMachine } from "./components/vsm/vsm";
+import { IStorageComponent } from "./components/storage/storage.interface";
+import { StorageComponent } from "./components/storage/storage";
+
```

Comment:
will need the prover modules here as well to select prover system etc. or is this part of the `VirtualStateMachine` module? @ognjenkurtic

Reply:
Yes,  they will be added here (or in the VSM, depending on our future decision) as part of upcoming [tasks](https://github.com/eea-oasis/baseline/projects/2) to integrate the zk mock